app/services/prototype.session.server.ts

```diff
@@ -29,7 +35,13 @@ export const createUserSession = async (
   session.set("accessToken", accessToken);
   session.set("expiresAt", expiresAt);
 
-  return commitSession(session);
+  try {
+    console.log("Creating user session with accessToken:", accessToken);
```

Comment:
Let's keep the `accessToken` out of our logs as this is sensitive data:
```suggestion
    console.log("Creating user session");
```